lib/hypervisor/hv_kvm/__init__.py

```diff
@@ -2467,7 +2453,25 @@ def _StopInstance(self, instance, force=False, name=None, timeout=None):
       else:
         self.qmp.Powerdown()
         if timeout is not None:
-          self._WaitForInstanceToShutdown(timeout, name)
+          tick = 1
+          while tick <= timeout:
```

Comment:
You know `timeout` here, you could replace `while tick <= timeout` with `for tick in range(1, timeout+1)`.